scripts/dts/edtlib.py

```diff
@@ -684,7 +685,7 @@ class Node:
       A list with the nodes that the node directly depends on
 
     enabled:
-      True unless the node has 'status = "disabled"'
+      True when the full path have 'status = "okay"', False otherwise.
```

Comment:
Could say

    True unless the node or one of its parents has 'status = "disabled"'`

Just to match the code.

Reply:
:+1: